src/inspect_ai/tool/__init__.py

```diff
@@ -23,6 +24,8 @@
     "python",
     "web_browser",
     "web_search",
+    "computer",
+    "computer_split",
```

Comment:
I think we should keep the `computer_split()` implementation private for now and just document `computer()`. We'll then just have the split version in our back pocket if some models struggle with the single version.